ipfx/lab_notebook_reader.py

```diff
@@ -67,7 +67,7 @@ def get_text_value(self, name, data_col, sweep_col, enable_col, sweep_num, defau
                     # this shouldn't happen, but if it does then bitch
                     #   as this situation hasn't been tested (eg, is
                     #   enabled indicated by 1.0, or "1.0" or "true" or ??)
-                    Exception("Enable flag not expected for text values")
+                    raise Exception("Enable flag not expected for text values")
```

Comment:
This should be a [more specific kind of exception](https://docs.python.org/3/library/exceptions.html#concrete-exceptions).